libs/components/datetime/src/lib/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -104,7 +104,7 @@ export class SkyDatepickerInputDirective
 
   /**
    * Specifies the latest date that is available in the calendar. Place this attribute on
-   * the `input` element to override the default in `SkyDatepickerConfigService`.
+   * the `input` element to override the default in `SkyDatepickerConfigService`. The specified date should be set to midnight in the current users timezone.
```

Comment:
A few things here:
- Saying that a "date" is "set to midnight" seems inaccurate. A date isn't set to a time, so I think we need to tweak this. I'd rewrite to make "time" the subject.
- "should be" makes it sound like this is a recommendation, but I think we're describing default behavior, not an optional recommendation. I'd rewrite to state this as something that happens, not something that should happen.
- "user's" should be possessive, not plural.
- "time zone" should be two words.

Here's a suggested update to incorporate my feedback, but I can try again if this doesn't hit the mark...
```suggestion
   * the `input` element to override the default in `SkyDatepickerConfigService`. The time on the specified date is set to midnight in the current user's time zone.
```